// Copyright (c) farm-ng, inc. All rights reserved.

syntax = "proto3";

package farm_ng_core.proto;

message Uri {
  string scheme = 1;
  string authority = 2;
  string path = 3;
  string query = 4;
}

message Timestamp {
  string clock_name = 1;
  double seconds = 2;
  string clock_type = 3;
}

message GeoSpherical {
  double longitude_rad = 1;
  double latitude_rad = 2;
}
